src/server/user.js

```diff
@@ -9,11 +10,18 @@ import Connection from 'server/core/connection';
  */
 export function createUser(newUser) {
   return new Promise((resolve, reject) => {
-    new Connection()
-      .post()
-      .users()
-      .data(newUser)
-      .call(resolve, reject);
+    if (NODE_ENV === 'development') {
+    import('server/core/utils/users.mock.js')
```

Comment:
I assume this is a stopgap until you have a back-end? If so, you might want to make some note of it